numpy/_globals.py

```diff
@@ -89,3 +92,19 @@ def __repr__(self):
 
 
 _NoValue = _NoValueType()
+
+class CopyMode(enum.Enum):
```

Comment:
This will need a `__module__` override, just like is used for all other public objects inside `_globals`.